cli/program_state.rs

```diff
@@ -120,7 +115,7 @@ impl ProgramState {
   pub async fn prepare_module_load(
     self: &Arc<Self>,
     specifier: ModuleSpecifier,
-    target_lib: TargetLib,
+    lib: TypeLib,
```

Comment:
Nitpick: use `type_lib`? A bit more descriptive than `lib`

Reply:
I have been trying my best to ensure consistent naming.  It is one of the things I found frustrating, is that things keep getting renamed as the move along.  This directly feeds the `lib` property in `TsConfig`, so I named it `lib`.  I agree it can bit a little bit less self documenting here, but if you follow it through the code, it is named the same everywhere you see it, instead of seeing `type_lib` in one place and `lib` in another. 🤷

Reply:
Fair enough, let's keep `lib`